public/app/core/components/RolePicker/RolePickerMenu.tsx

```diff
@@ -507,6 +512,10 @@ const capitalize = (s: string): string => {
 
 const sortRolesByName = (a: Role, b: Role) => a.name.localeCompare(b.name);
 
+const isDelegatable = (role: Role) => {
```

Comment:
I think this does the opposite of its name? 🤔

afaict
Not set: false (should be true — since that means the filtering isn't done on the backend)
Set to true: false (should be true)
Set to false: true (should be false)

Reply:
Ah, true!